### PR TITLE
Jan migrate kbc styles pt3 

### PR DESCRIPTION
**Pull request by janmichek**

### CODE REVIEW

File: src/indigo/less/shame.less [hunk 1]
```diff
@@ -370,3 +362,163 @@ small.empty-label {
...
+  padding-left: 28px;
+  margin-top: 10px;
+}
+
+.kbc-sidebar-jobs .jobs-link a {
+  color: #7e8490;
+}
+
+
+//moved from AuthorizationModal.less
```
Comment (ujovlado) [inline src/indigo/less/shame.less]: @janmichek znamenta to ze `.kbc-authorization-modal` ide prec kompletne? (tam bol ten wtf fix)

Comment (janmichek) [reply src/indigo/less/shame.less]: Ano, wtf fix uz je k nicemu 
image

potom uz ta classa '.kbc-authorization-modal' bude k nicemu. Specifikovala jenom wtf fix

Comment (janmichek) [reply src/indigo/less/shame.less]: image

Comment (ujovlado) [reply src/indigo/less/shame.less]: ok, potom ok :)

**Pull request merged**